velox/functions/sparksql/RegisterArithmetic.cpp

```diff
@@ -35,7 +35,8 @@ void registerArithmeticFunctions(const std::string& prefix) {
   registerBinaryNumeric<PlusFunction>({prefix + "add"});
   registerBinaryNumeric<MinusFunction>({prefix + "subtract"});
   registerBinaryNumeric<MultiplyFunction>({prefix + "multiply"});
-  registerFunction<DivideFunction, double, double, double>({prefix + "divide"});
+  registerFunction<DoubleDivideFunction, double, double, double>({prefix + "divide"});
+  registerFunction<LongDivideFunction, long, long, long>({prefix + "divide"});
```

Comment:
I think only one template function is needed, seems these two functions has no differences in implementation.